sections/product-new-template.liquid

```diff
@@ -0,0 +1,500 @@
+{% comment %}
```

Comment:
It's impossible to tell in github but make sure the indentation is uniform on this.